mappings/net/minecraft/client/render/entity/model/ShulkerEntityModel.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_602 net/minecraft/client/render/entity/model/ShulkerEntityModel
+	FIELD field_3553 bottomShell Lnet/minecraft/class_630;
+	FIELD field_3554 center Lnet/minecraft/class_630;
```

Comment:
```suggestion
	FIELD field_3554 head Lnet/minecraft/class_630;
```
https://minecraft.gamepedia.com/Shulker#Behavior calls it a `head`, which I think makes sense.

Reply:
👍